src/react-components/ui-root.js

```diff
@@ -1643,7 +1663,8 @@ class UIRoot extends Component {
                 >
                   {!embed &&
                     !showVREntryButton &&
-                    !hasTopTip && (
+                    !hasTopTip &&
+                    !this.state.isStreaming && (
```

Comment:
since the logic here has gotten haywire, i've been putting these in vars, i'd suggest `streaming`